### DISCUSSION TITLE
MRIS_ANATOMICAL_STATS w/ multiple subjects

### DISCUSSION THREAD

**Discussion by clarefmccann** [now-i-know-my-abcd/docs#35] [Stats & Code Help]
Has anyone run freesurfer's `mris_anatomical_stats` for a non-freesurfer-generated ROI on multiple subjects to have the output be one text file, similar to when you run `aparcstats2table`?

**Comment by JunhongYu** [comment]
Not sure if this helps, but i believe the columns in abcd_mrisdp10201.txt are arranged in a similar manner as those in the aparcstats2table output files.  In anycase the freesurfer stats folder, which are required for running aparcstats2table can be downloaded from https://collection3165.readthedocs.io/en/stable/derivatives/#freesurfer-statistics-derivativesanatstats
